### PR TITLE
chore:setup newrelic

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Change
setup newrelic

## Evidence
title: setup newrelic
project: Jabar Super Apps
participants: @marsellavaleria19 @Ibwedagama @rizfirman @agunghide @naufalihsank @yoslie @doohanas @rayfajars

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@nuxtjs/svg": "^0.4.0",
     "@sentry/tracing": "^6.19.7",
     "core-js": "^3.19.3",
+    "newrelic": "^9.9.0",
```
Comment (Ibwedagama) [inline package.json:18]: hello, @marsellavaleria19 should we remove sentry from the package list?

**Review by Ibwedagama — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

**Review by rizfirman — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Comment by rayfajars** [comment]
add secret on file staging.yaml , if want deploy staging new relic

**Review by rayfajars — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (marsellavaleria19) [reply package.json:18]: Hello, @Ibwedagama .. Yes, we should remove this. But, i forgot to remove this.  Thank you for reminding me. This is my commit : https://github.com/jabardigitalservice/super-app-web/pull/33/commits/75b23933ef2f730f6e6ffe25d22ec968c438247b

**Comment by marsellavaleria19** [comment]
Hello @rayfajars.. Thank you for reminding me.  I've ix it. This is my commit : https://github.com/jabardigitalservice/super-app-web/pull/33/commits/9a42b403d92c929107782632b01bc943ed220c41

**Pull request merged**